.changeset/old-wombats-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@preconstruct/cli": patch
+---
+
+Add `newDistFilenames` experimental flag
```

Comment:
this PR has added `distFilenameStrategy` as well, probably worth mentioning here

Reply:
`distFilenameStrategy` is behind the `newDistFilenames` flag, will mention it in the changeset that unflags it

Reply:
oh ok - i've missed this when skimming through the implementation, wouldn't it make more sense to have this under experimental key?

Reply:
The intent is for distFilenameStrategy to be a very well-supported API(once newDistFilenames is unflagged in v2) since people might not want to change their dist filenames for a while/ever

Reply:
Sure, **once** it `newDistFilenames` gets unflagged - then this makes sense to me, but at the moment it's somewhat confusing to have a top-level option that is only enabled with an experimental flag. Not a big deal - but this has caught my completely off guard.